native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.android.js

```diff
@@ -0,0 +1,255 @@
+  TouchableWithoutFeedback,
+  View,
+  ViewPropTypes,
+} from 'react-native';
+
+import {
+  borderRadiusSm,
+  colorBlue500,
+  colorGray50,
+  colorGray500,
+  colorGreen500,
+  colorRed500,
+  colorYellow500,
+  spacingBase,
+  spacingLg,
+  spacingMd,
+  spacingSm,
+  spacingXxl,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import BpkAnimateHeight from 'react-native-bpk-component-animate-height';
+import BpkIcon from 'react-native-bpk-component-icon';
+import BpkText from 'react-native-bpk-component-text';
+
+import AnimateAndFade from './AnimateAndFade';
+import {
+  dismissablePropType,
+  toggleExpandedButtonLabelPropType,
+  dismissableLabelPropType,
+} from './customPropTypes';
+
+export const ALERT_TYPES = {
+  SUCCESS: 'success',
+  WARN: 'warn',
+  ERROR: 'error',
+  NEUTRAL: 'neutral',
+};
+
+const styles = StyleSheet.create({
+  mainContainer: {
+    paddingVertical: spacingBase,
+    paddingHorizontal: spacingLg,
+    backgroundColor: colorGray50,
+    minHeight: spacingXxl + spacingBase,
+    flex: 1,
+  },
+  dismissableContainer: {
+    paddingHorizontal: spacingLg - spacingMd,
+    backgroundColor: colorGray50,
+    justifyContent: 'center',
+  },
+  childrenContainer: {
+    paddingVertical: spacingSm,
+  },
+  flexRow: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  text: {
+    flex: 1,
+  },
+  icon: {
+    paddingRight: spacingSm,
+    paddingTop: spacingSm - spacingSm / spacingSm,
+  },
+  dismissLabel: {
+    color: colorBlue500,
+  },
+  dismissButtonContainer: {
+    padding: spacingMd,
+    borderRadius: borderRadiusSm,
+  },
+  dismissButtonContainerRadius: {
+    borderRadius: borderRadiusSm,
+  },
+});
+
+const ALERT_TYPE_STYLES = {
+  [ALERT_TYPES.SUCCESS]: {
+    iconSource: 'tick-circle',
+    iconStyle: {
+      color: colorGreen500,
+    },
+  },
+  [ALERT_TYPES.WARN]: {
+    iconSource: 'information-circle',
+    iconStyle: {
+      color: colorYellow500,
+    },
+  },
+  [ALERT_TYPES.ERROR]: {
+    iconSource: 'information-circle',
+    iconStyle: {
+      color: colorRed500,
+    },
+  },
+  [ALERT_TYPES.NEUTRAL]: {
+    iconSource: 'information-circle',
+    iconStyle: {
+      color: colorGray500,
+    },
+  },
+};
+
+const BpkBannerAlert = props => {
+  const {
+    type,
+    message,
+    onDismiss,
+    onToggleExpanded,
+    dismissable,
+    expanded,
+    dismissButtonLabel,
+    toggleExpandedButtonLabel,
+    show,
+    style: userStyle,
+    animateOnEnter,
+    animateOnLeave,
+    children,
+    ...rest
+  } = props;
+  const expandable = children !== null;
+  const { iconSource, iconStyle } = ALERT_TYPE_STYLES[type] || {};
+
+  const dismissableButton = () => {
+    if (!dismissable) {
+      return null;
+    }
+    return (
+      <View style={styles.dismissableContainer}>
+        <View style={styles.dismissButtonContainerRadius}>
+          <TouchableNativeFeedback
+            accessibilityComponentType="button"
+            accessibilityLabel="dismiss"
```

Comment:
This looks like an error. This should be coming from props rights?

Reply:
Woah, nice catch!! 🦅